layouts/index.html

```diff
@@ -8,7 +8,7 @@
 
     <h1>{{ .Site.Title }}</h1>
     {{ if isset .Site.Params "description" }}
-        <h3>{{ .Site.Params.Description }}</h3>
+        <h2>{{ .Site.Params.Description }}</h2>
```

Comment:
This increases the font size of the about description. I tried and setting `font-size: 1.2em;` makes it back to normal. Can you please add the css rule?
```css
.home-about h2 {
  font-size: 1.2em;
}
```

Reply:
Before: 

![image](https://github.com/user-attachments/assets/cfb99ef6-d12b-4b07-a652-6e8532d964e4)

After: 

![image](https://github.com/user-attachments/assets/4be9c0bf-7d92-418e-ac94-09344bb36328)